src/app/components/elements/layout/SidebarLayout.tsx

```diff
@@ -494,12 +494,19 @@ export const MyAccountSidebar = (props: SidebarProps) => {
 
 export const SignupSidebar = ({activeTab} : {activeTab: number}) => {
     const history = useHistory();
+
+    const goBack = (path: string) => {
+        confirmThen(
+            "Are you sure you want go back? Any information you have entered will be lost.",
+            () => history.push(path));
+    };
+
     return <ContentSidebar buttonTitle="Create an account">
         <div className="section-divider mt-4"/>
         <h5 className="mt-1">Create an account</h5>
         {/* Tabs are clickable iff their page could be reached with a Back buttons */}
-        <StyledTabPicker checkboxTitle={"Sign-up method"} checked={activeTab === 0} disabled={activeTab > 2} onClick={activeTab > 0 && activeTab < 3 ? (() => history.push("/register")) : undefined}/>
-        <StyledTabPicker checkboxTitle={"Age verification"} checked={activeTab === 1}  disabled={activeTab < 1 || activeTab > 2} onClick={activeTab === 2 ? (() => history.push("age")) : undefined}/>
+        <StyledTabPicker checkboxTitle={"Sign-up method"} checked={activeTab === 0} disabled={activeTab > 2} onClick={() => goBack("/register")}/>
+        <StyledTabPicker checkboxTitle={"Age verification"} checked={activeTab === 1}  disabled={activeTab < 1 || activeTab > 2} onClick={() => goBack("age")}/>
```

Comment:
`onClick` still works if the element is disabled, so if the disabled condition is met we shouldn't run `goBack`. We probably also don't want it to do anything if you "switch" to the currently-active tab.
```suggestion
        <StyledTabPicker checkboxTitle={"Sign-up method"} checked={activeTab === 0} disabled={activeTab > 2} onClick={() => (activeTab === 1 || activeTab === 2) && goBack("/register")}/>
        <StyledTabPicker checkboxTitle={"Age verification"} checked={activeTab === 1} disabled={activeTab < 1 || activeTab > 2} onClick={() => activeTab === 2 && goBack("age")}/>
```